### DISCUSSION TITLE
MacOS

### DISCUSSION THREAD

**Discussion by sarbjitbilling** [rcaelers/workrave#313] [General]
Hi - Will a MacOS version of WorkRav be released and in the future?

**Comment by rcaelers** [comment]
I am working on a macOS version using Qt. There is still some work to do before it is ready.

<img width="978" alt="Screenshot 2021-08-01 at 00 09 04" src="https://user-images.githubusercontent.com/80881/127753460-9acbd533-a66c-4643-9244-2b90c0853e3c.png">

<img width="755" alt="Screenshot 2021-08-01 at 00 12 28" src="https://user-images.githubusercontent.com/80881/127753535-bcc1a54f-31b6-4539-87ed-3c6f7a07efa5.png">

**Reply by rlue** [reply]
I just got workrave set up on my Linux machine and it's a godsend! Really looking forward to the day it's available on macOS too.

Any chance the dev version is in a semi-usable state? I'd be happy to build from source, but have had a hard time finding up-to-date build instructions.

**Comment by binp** [comment]
What is the current status of macos version? It looks like the qt has been submitted in the main branch. Are there any other tasks to be done before any macos release?
